apps/demo-app/src/app/examples/button/button-example.module.ts

```diff
@@ -2,26 +2,17 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { ButtonExampleComponent } from './button-example/button-example.component';
 import { SharedModule } from '../../shared/shared.module';
-import { ENTRY_BUTTON_CONFIG, EntryButtonConfig, EntryButtonModule } from '@enigmatry/entry-components/button';
 
 @NgModule({
   declarations: [
     ButtonExampleComponent
   ],
   imports: [
     CommonModule,
-    SharedModule,
-    EntryButtonModule
+    SharedModule
   ],
   exports: [
     ButtonExampleComponent
-  ],
```

Comment:
The example is incomplete without this code.

Reply:
I agree and disagree in the same time. We need to have definition for buttons in shared module since that's optimal and representative; developers will look at this app as example for many best practices, that's why we can't allow inline styles and other crap.
On the other side I partially agree since in theory we can use these buttons independently, but there's not too much sense in that if we only support submit and cancel button. Whenever we have those buttons we usually (should) have a form and ideally generated one which already includes buttons.